cli/index.js

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env node
 const cli = require("caporal");
 const { OPTION_DEFAULTS: SSR_OPTION_DEFAULTS, run: runSSR } = require("ssr");
+const {
+  writeContentVersion,
+  checkContentVersion,
+  VersionStatus
+} = require("./content-version");
```

Comment:
I found the exported function names to be expressive enough that namespacing them wouldn't have helped with that. Then again namespacing does make it easier to grok that they come from the same module. I'm torn with a bias towards status-quo

Reply:
If it was Python I would not be torn. Not just because of this:
```
python
>>> import this
The Zen of Python, by Tim Peters
...
Namespaces are one honking great idea -- let's do more of those!
```
But I can admit that there are times when importing classes/functions directly instead of the module has advantages. 

The thing that always makes me nervous in JS code is if the tree-shaking is going to fail me if I import the whole thing. 
E.g. `import lodash from 'lodash'` is much worse than `import { merge } from 'lodash'` but that one sucks too because what was your brain think when it sees `merge` somewhere in a 1,000 line .js file?? 

The problem with your code now is that `VERSION_FILE_PATH` loses it's context completely. "Which VERSION?? What's its context??" you think when you're reading code far far away from line 8. 

But, for `boxify` it makes sense. It'd be a shame to have to type `boxify.boxify('Bla bla')`.

Reply:
I'm not trying to convince you but I often feel the need to type something like this:
```
const repository = await nodegit.repository(PATH)
```
Yes, I know it's actually called `Repository` but often the main export would be a great name of an instance of it. 

I just generally think it's a fact of life in JS that you simply have to learn to live with it. It's as common as 2-space indentation that you import the functions rather than the modules. It's just how it's often done and how you often see it in documentation and code snippets.

Reply:
I do appreciate that perspective though I don't share it. But I also don't think I'm right, my mind just works different when looking at stuff like this. For the example of `VERSION_FILE_PATH`, when I skim through the file and see line where it's used I don't think I'm that interested where it comes from as long as the name tells me what it roughly is. Now if I am interested where it comes from I'd jump to its declaration (which I suppose not all editors support, so my tools might inform my code style here).

I do take the general point that for names which are ambigiuos in that context, namespacing can help disambiguating it, which I see somewhat for `VERSION_FILE_PATH` (less so for `Repository`). Apart from that I opt for brevity.

Reply:
We're nerds. It's OK to have "debates" so we can both grow. I definitely feel like there's a difference on the general preferences that is different depending on the language. I.e. python vs js. 
And there's definitely arguments on both sides. 

I'm definitely not trying to convince you but one argument for namespacing is that when I'm *in* the file `some-xyz-feature.js` I can write:
```
export function run() {
   ...
}
function think() { ... }
function calculate() { ... }
```
...without having to worry about someone doing `import { run } from './some-xyz-feature.js'` inside a 1,000 lines file. 
When I'm inside `some-xyz-feature.js` the name `run` makes perfect sense. And it's an OK name if the file `some-xyz-feature.js` isn't massive. 

Also, with namespacing I can now start working on `other-abc-feature.js` and do a `export function run() {` in that one too without having to worry about how it's going to be used.

Reply:
Did some Rust over the weekend and now my view is more in-line with yours. In that ecosystem they might also rename the three exports in this case to `write`, `check` and `Status`. And I think I'm getting more used to that idea. 
I'll take that idea for a ride!